### PR TITLE
feat(IT Wallet): [SIW-289] Add cie info screen on activation flow

### PR DESCRIPTION
**Pull request by hevelius**
## Short description
This PR adds an info screen on activation flow

## List of changes proposed in this pull request
- Add new info screen on activation flow
- Update locales

## How to test
Start activation flow and just after pin screen an info screen should be viewed.

### CODE REVIEW

File: ts/features/it-wallet/screens/issuing/cie/CieInfoUsageScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+          <View style={IOStyles.horizontalContentPadding}>
+            <H4 weight={"Regular"} color={"bluegrey"}>
+              <Link
+                onPress={() =>
+                  openWebUrl(
+                    I18n.t("features.itWallet.issuing.infoCie.readMoreUrl")
+                  )
+                }
+              >
+                {"Guarda come fare"}
```
Comment (LazyAfternoons) [inline ts/features/it-wallet/screens/issuing/cie/CieInfoUsageScreen.tsx:80]: Maybe we should add this to the translation file as well?

Comment (hevelius) [reply ts/features/it-wallet/screens/issuing/cie/CieInfoUsageScreen.tsx:80]: Yep, thx!

File: ts/features/it-wallet/screens/issuing/cie/CieInfoUsageScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  const renderContent = () => {
+    const cancelButtonProps = {
+      block: true,
+      light: false,
+      bordered: true,
+      onPress: () =>
+        navigation.navigate(ROUTES.MAIN, {
+          screen: ROUTES.MESSAGES_HOME
+        }),
+      title: I18n.t("features.itWallet.activationScreen.cancel")
```
Comment (LazyAfternoons) [inline ts/features/it-wallet/screens/issuing/cie/CieInfoUsageScreen.tsx:48]: I think this one should be `title: I18n.t("features.itWallet.issuing.infoCie.cancel")`, what do you think?

Comment (hevelius) [reply ts/features/it-wallet/screens/issuing/cie/CieInfoUsageScreen.tsx]: Fixed https://github.com/pagopa/io-app/pull/4783/commits/9418fbaf23fb3bb99f8e31aa55aad034e8d616ec

**Pull request merged**